pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataSetup.java

```diff
@@ -93,7 +92,7 @@
         private String globalZookeeper;
 
         @Parameter(names = { "-cs",
-            "--configuration-store" }, description = "Configuration Store connection string", required = false)
+            "--configuration-store" }, description = "Configuration Store connection string", required = true)
```

Comment:
Hi @murong00,  Is this change necessary?

Reply:
If someone missed this option, it will print error `Configuration store address argument is required (--configuration-store)` since `--global-zookeeper` option is hidden . So it may be better to make this option `required`.